src/python/pants/backend/awslambda/python/rules_test.py

```diff
@@ -207,26 +207,26 @@ def handler(event, context):
     ("ics", "runtime", "architecture"),
     [
         pytest.param(
-            ["==3.7.*"],
+            ["==3.8.*"],
             None,
             FaaSArchitecture.X86_64,
             id="runtime inferred from ICs, x86_64 architecture",
         ),
         pytest.param(
             None,
-            "python3.7",
+            "python3.8",
             FaaSArchitecture.X86_64,
             id="runtime explicitly set, x86_64 architecture",
         ),
         pytest.param(
-            ["==3.7.*"],
+            ["==3.8.*"],
```

Comment:
Note - this change was made because AWS doesn't support ARM for Python 3.7, and removing the fallback logic for platform resolution made this test fail. I consider that a win :)